src/tests/test_piranha_go.rs

```diff
@@ -12,15 +12,15 @@ Copyright (c) 2022 Uber Technologies, Inc.
 */
 
 use super::{
-  check_result, copy_folder, create_match_tests, create_rewrite_tests, initialize, substitutions,
+  copy_folder_to_temp_dir, create_match_tests, create_rewrite_tests,
+  execute_piranha_and_check_result, initialize, substitutions,
```

Comment:
We add `execute_piranha_and_check_result`, but we aren't actually using it in this file for now, are we?

Reply:
This is because the `create_rewrite_tests` macro expands to use the `execute_piranha_and_check_result` method.